web/viewer.html

```diff
@@ -142,10 +142,12 @@
               <span data-l10n-id="find_next_label">Next</span>
             </button>
           </div>
-          <input type="checkbox" id="findHighlightAll" class="toolbarField">
```

Comment:
The idea behind the new div was that it kept the checkboxes together as the screen size got smaller. It's not very useful to have a checkbox on one line and its label on another. The id was there for clarity. If there is a better solution< i'm happy to hear it.